vehicle/mkv/software/throttle/throttle_config.h

```diff
@@ -0,0 +1,70 @@
+#include "libs/adc/api.h"
+#include "libs/gpio/api.h"
+#include "libs/gpio/pin_defs.h"
+
+/*
+ * Pin definitions
+ */
+
+gpio_t BRAKE_IMPLAUSIBILTIY_LED = PC6;
+gpio_t DEVIATION_IMPLAUSIBILITY_LED = PB3;
+gpio_t OUT_OF_RANGE_IMPLAUSIBILITY_LED = PB4;
+
+gpio_t RJ45_LED1 = PB0; // orange
+gpio_t RJ45_LED2 = PB1; // green
+
+gpio_t SS_ESTOP = PB5;
+gpio_t SS_IS = PB6;
+gpio_t SS_BOTS = PB7;
+
+/*
+ * ADC pins
+ */
+// adc_pin_e DRIVE_MODE_SENSE = ADC2; // UNUSED
+
+/*
+ * Represents a throttle potentiometer, or APPS (Accelerator Pedal Position
+ * Sensor)
+ */
+typedef struct {
+    adc_pin_e adc_pin;
+    uint16_t throttle_min; // Minimum position (0%) in raw ADC
+    uint16_t throttle_max; // Maximum position (100%) in raw ADC
+} throttle_potentiometer_s;
+
+// Define throttle values to use when mapping
+// Values are capped at this range
+// TODO: Recheck/set these values
+#define THROTTLE_R_MIN_COUNTS (uint16_t)(218)
+#define THROTTLE_R_MAX_COUNTS (uint16_t)(654)
+#define THROTTLE_L_MIN_COUNTS (uint16_t)(298)
+#define THROTTLE_L_MAX_COUNTS (uint16_t)(1016)
+
+const throttle_potentiometer_s throttle_r = {
+    .adc_pin = ADC8,
+    .throttle_min = THROTTLE_R_MIN_COUNTS,
+    .throttle_max = THROTTLE_R_MAX_COUNTS,
+};
+
+const throttle_potentiometer_s throttle_l = {
+    .adc_pin = ADC9,
+    .throttle_min = THROTTLE_L_MIN_COUNTS,
+    .throttle_max = THROTTLE_L_MAX_COUNTS,
+};
+
+/*
+ * Timer config
+ */
+void timer0_isr(void);
```

Comment:
why are we setting up the timer if it's ISR is empty? Are we using this to time CAN messages?

Reply:
This is just the function prototype, so that we can pass the reference to the struct (basically lets us pass a pointer to the function in the struct), and then we actually _define_ the function in `throttle.c`.

The `timer0_isr` sets a variable called `send_can = true` and triggers every ~10ms, so yeah 100Hz.